### PR TITLE
Meeting 14 April, 2020

### PR DESCRIPTION
**Pull request by Tommoa**

### CODE REVIEW

File: 2020/2020-04-14.committee.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+
+Absent:
+- Jordan
+
+## President's report
+- Start writing some Agile development for the book (not committed until first draft)
+- Helped Dadams write some slides
+- Applied for Google things
+- Various admin things
+- Research is now going well
```
Comment (gozzarda) [inline 2020/2020-04-14.committee.md:23]: Add spacing after reports?

File: 2020/2020-04-14.committee.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+    against your notebook
+- What happens if you loudly talk about solving problems with DP?
+    - A fun time with Gozz and Max
+- Albert has apparently moved to being a physics major
+- Code Jam is two words, not one word
+    - So is Kick Start
+- Guild Finance does a dumb thing where they match deductions against additions
+    - The format exported from NetSuite is really bad
+    - Gozz has a way to deal with them [here](https://netsuite.gozz.id.au)
+        - It's <ins>**Open Source**</ins>(tm)!
```
Comment (gozzarda) [inline 2020/2020-04-14.committee.md:199]: Why the ins?

File: 2020/2020-04-14.committee.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+    - Have something that runs line by line with the preprocessor and a cumulative hash to check
+    against your notebook
+- What happens if you loudly talk about solving problems with DP?
+    - A fun time with Gozz and Max
+- Albert has apparently moved to being a physics major
+- Code Jam is two words, not one word
+    - So is Kick Start
+- Guild Finance does a dumb thing where they match deductions against additions
+    - The format exported from NetSuite is really bad
+    - Gozz has a way to deal with them [here](https://netsuite.gozz.id.au)
```
Comment (gozzarda) [inline 2020/2020-04-14.committee.md:198]: Could this not just be a bare link?

File: 2020/2020-04-14.committee.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
+# UWA PCS Committee Meeting Agenda 2020-04-14
```
Comment (gozzarda) [inline 2020/2020-04-14.committee.md:1]: Consider adding "Opening" section for opening time and attendance for good document structure?

File: 2020/2020-04-14.committee.md (hunk 2 above)
Comment (Tommoa) [reply 2020/2020-04-14.committee.md:199]: to add underline

File: 2020/2020-04-14.committee.md (hunk 3 above)
Comment (Tommoa) [reply 2020/2020-04-14.committee.md:198]: yes, but hey

File: 2020/2020-04-14.committee.md (hunk 4 above)
Comment (Tommoa) [reply 2020/2020-04-14.committee.md:1]: Good idea

**Pull request merged**